xbmc/pvr/PVRManager.cpp

```diff
@@ -138,6 +136,30 @@ CPVRManager &CPVRManager::GetInstance()
   return CServiceBroker::GetPVRManager();
 }
 
+CPVRChannelGroupsContainerPtr CPVRManager::ChannelGroups(void) const
+{
+  CSingleLock lock(m_critSection);
+  return m_channelGroups;
+}
+
+CPVRRecordingsPtr CPVRManager::Recordings(void) const
+{
+  CSingleLock lock(m_critSection);
+  return m_recordings;
+}
+
+CPVRTimersPtr CPVRManager::Timers(void) const
+{
+  CSingleLock lock(m_critSection);
+  return m_timers;
+}
+
+CPVRClientsPtr CPVRManager::Clients(void) const
+{
+  // note: m_addons is const (only set/reset in ctor/dtor). no need for a lock here.
```

Comment:
It looks like the lock we discussed earlier got removed? Or is this intentional?